backend/src/services/auth/auth.service.ts

```
@@ -651,36 +652,158 @@ export class AuthService {
+      await client.query('ROLLBACK');
+      throw error;
+    } finally {
+      client.release();
+    }
+  }
+
+  //get all admins
+  async getAllAdmins(): Promise<ListUsersResponse> {
+    const dbManager = DatabaseManager.getInstance();
+    const pool = dbManager.getPool();
+    const result = await pool.query(`SELECT id, username, created_at, updated_at FROM auth.project_admins WHERE created_by IS NOT NULL`);
```

ESLint

[error] Replace ``SELECT·id,·username,·created_at,·updated_at·FROM·auth.project_admins·WHERE·created_by·IS·NOT·NULL`` with `⏎······`SELECT·id,·username,·created_at,·updated_at·FROM·auth.project_admins·WHERE·created_by·IS·NOT·NULL`⏎····`

(prettier/prettier)